src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/providers/NotcreatedDBInfluxDB.java

```diff
@@ -0,0 +1,230 @@
+package com.rackspacecloud.metrics.ingestionservice.influxdb.providers;
+
+
+import okhttp3.OkHttpClient;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.influxdb.dto.Query;
+import org.influxdb.dto.QueryResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.NumberUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class NotcreatedDBInfluxDB {
+
+    private InfluxDB influxDB;
+
+    float bkuprowcountval;
+    float resrowcountval;
+    List<Float> al=new ArrayList<Float>();
+    public List<Object> bk=new ArrayList<Object>();
+    List<Object> bkrwcnt=new ArrayList<Object>();
+    List<Object> resrwcnt=new ArrayList<Object>();
+
+    public  InfluxDB influxDbBuild()
+    {
+        InfluxDB influxDB = null;
+        OkHttpClient.Builder client = new OkHttpClient.Builder()
+                .connectTimeout(1, TimeUnit.MINUTES)
+                .readTimeout(1, TimeUnit.MINUTES)
+                .writeTimeout(1, TimeUnit.MINUTES)
+                .retryOnConnectionFailure(true);
```

Comment:
In other Ceres code (and when I have used InfluxDB), we don't bother customizing the HTTP client. Was there a reason to not use default setup or was it just a pattern followed from elsewhere?